org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/imports/ImportOrganizer.java

```diff
@@ -56,7 +57,16 @@
 		if (unresolvedTypeResolver != null)
 			unresolvedTypeResolver.resolve(typeUsages, resource);
 		Map<String, JvmDeclaredType> name2type = conflictResolver.resolveConflicts(typeUsages, nonOverridableTypesProvider, resource);
-		return getOrganizedImportChanges(resource, name2type, typeUsages);
+		List<ReplaceRegion> changes = getOrganizedImportChanges(resource, name2type, typeUsages);
```

Comment:
This looks fishy to me. Modifying the returned list looks not right and a method called 'getXYZChanges' should probably not return things that are no changes at all? Would it be too cumbersome to avoid these replaceRegions in the returned list in the first place?